src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i in actions) {
+    switch (actions[i].type) {
```

Comment:
```suggestion
    switch (action.type) {
```
Also, you can make destructuring to get all the needed properties of action.